.github/workflows/benchmark-pipeline.yaml

```diff
@@ -51,6 +51,8 @@ jobs:
         run: |
           kubectl apply -f manifest.yaml
 
+          sleep 10
```

Comment:
Without the sleep the wait command fails with `error: no matching resources found` 

Not the most graceful of fixes 🔨  but it works 😄